### PR TITLE
implement option to continue if time integration failed with minimum time step

### PR DESCRIPTION
**Pull request by andlaus**
this is basically a continuation of #522. By itself, this PR does nothing if `--continue-on-convergence-error=true` is not explicitly specified on the command line.

### CODE REVIEW

**Comment by andlaus** [comment]
jenkins build this please

File: ewoms/disc/common/fvbaseproblem.hh [hunk 1]
```diff
@@ -521,10 +525,14 @@ public:
             Scalar dt = simulator().timeStepSize();
             Scalar nextDt = dt / 2.0;
             if (dt < minTimeStepSize*(1 + 1e-9)) {
-                errorMessage =
-                    "Time integration did not succeed with the minumum time step size of "
-                    + std::to_string(double(minTimeStepSize)) + " seconds. Giving up!";
-                break; // give up: we can't make the time step smaller anymore!
+                if (asImp_().continueOnConvergenceError())
+                    return;
```
Comment (totto82) [inline ewoms/disc/common/fvbaseproblem.hh]: I suggest adding a warning here.

**Comment by totto82** [comment]
Thanks. That was fast.

File: ewoms/disc/common/fvbaseproblem.hh (hunk 1 above)
Comment (andlaus) [reply ewoms/disc/common/fvbaseproblem.hh]: good point. added...

**Comment by andlaus** [comment]
jenkins build this please

**Pull request merged**